src/views/dialogs.html

```diff
@@ -29,6 +29,11 @@
                         <input type="text" ng-model="importNameColumn" size="3">
                         ({{importNameExample}})
                     </p>
+                    <p>
+                        Use custom delimiter? (Default is tab, the delimiter used when pasting from excel) <br/>
+                        <input type="checkbox" ng-model="useCustomDelimiter">
+                        <input type="text" ng-model="delimiter">
```

Comment:
This box should be disabled if `useCustomDelimiter` is false.
Or, make it a radio-button with `Tab (Excel)` as first option, and `Custom [input field]` as second?